app/assets/sass/application.scss

```diff
@@ -13,3 +13,19 @@ $govuk-global-styles: true;
 @import "components/cookie-banner";
 
 // Add extra styles here, or re-organise the Sass files in whichever way makes most sense to you
+
+.wide .govuk-summary-list__row .govuk-summary-list__key {
+  width: 68% !important;
```

Comment:
I often find using `!important` just leads to headaches later down the line when you can't figure out why some style isn't applying. Could we use a more specific selector here so it's not needed?